test-cloud/test-cloud-remote/src/test/resources/test.properties

```diff
@@ -26,5 +26,5 @@ amq.password=usetheforce123@
 kie.artifact.version=please-define-kie-artifact-version
 kjars.build.settings.xml=src/test/resources/settings.xml
 
-rhba.images.branch=master
+rhba.images.branch=7.7.x
```

Comment:
same here with setting it back to master

Reply:
My bad. Reverted